src/Sylius/Bundle/ImportExportBundle/DependencyInjection/Compiler/RegisterReadersPass.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ImportExportBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * Registers all readers in export profile registry service.
+ *
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class RegisterReadersPass implements CompilerPassInterface
```

Comment:
Let's stick to our current convention. If we decide that it is wrong, then we change all compiler passes at once.